src/sage/matroids/chow_ring_ideal.py

```diff
@@ -0,0 +1,643 @@
+            I_{FY} of matroid \text{\texttt{Graphic{ }matroid{ }of{ }rank{ }2{ }on{ }3{ }elements}}
+        """
+        from sage.misc.latex import latex
+        return 'I_{FY} of matroid ' + (latex(self._matroid))
+
+    def groebner_basis(self, algorithm='', *args, **kwargs):
+        r"""
+        Return the Groebner basis of ``self``.
+
+        EXAMPLES::
+
+            sage: ch = matroids.Uniform(2,5).chow_ring(QQ, True, 'fy')
+            sage: ch.defining_ideal().groebner_basis(algorithm='')
+            Polynomial Sequence with 565 Polynomials in 12 Variables
+            sage: ch.defining_ideal().groebner_basis(algorithm='').is_groebner()
+            True
+        """
+        if algorithm == '':
+            algorithm = 'constructed'
+        if algorithm != 'constructed':
+            return super().groebner_basis(algorithm=algorithm, *args, **kwargs)
+        gb = [] #Reduced groebner basis with two eliminated cases
+        E = list(self._matroid.groundset())
+        poly_ring = self.ring()
+        for F in self._flats:
+            for G in self._flats:
+                if not (F <= G or G <= F): #Non-nested flats
+                        gb.append(self._flats_generator[F]*self._flats_generator[G])
+
+                for i in E:
+                    term = poly_ring.zero()
+                    term1 = poly_ring.zero()
+                    for H in self._flats:
+                        if i in H:
+                            term += self._flats_generator[H]
+                        if H >= G:
+                            term1 += self._flats_generator[H]
+                    if term != poly_ring.zero():
+                        gb.append(self._flats_generator[i] + term) #5.7
+                    if term1 != poly_ring.zero():
+                        gb.append(term1**(self._matroid.rank(G) + 1)) #5.6
+
+                    if G > F: #nested flats
+                        if term1 != poly_ring.zero():
+                            gb.append(self._flats_generator[F]*term1**(self._matroid.rank(G)-self._matroid.rank(F)))
+
+        g_basis = PolynomialSequence(poly_ring, [gb])
+        return g_basis
+
+class AugmentedChowRingIdeal_atom_free(ChowRingIdeal):
+    r"""
+    The augmented Chow ring ideal for a matroid `M` over ring `R` in the
+    atom-free presentation.
+
+    The augmented Chow ring ideal in the atom-free presentation for a matroid
+    `M` is defined as the ideal `I_{af}(M)` of the polynomial ring:
+
+    .. MATH::
+
+        R[x_{F_1}, \ldots, x_{F_k}],
+
+    where
+
+    - `F_1, \ldots, F_k` are the non-empty flats of `M`,
+    - `I_{af}(M)` is the ideal generated by all quadratic forms `x_{F_i} x_{F_j}`
+      where `F_i` and `F_j` are incomparable elements in the lattice of flats,
+
+      .. MATH::
+
+        x_F \sum_{i \in F'} x_{F'}
+
+      for all `i \in E` and `i \notin F`, and
+
+      .. MATH::
+
+        \sum_{i \in F'} (x_{F'})^2
+
+      for all `i \in E`.
```

Comment:
```suggestion
    where `F_1, \ldots, F_k` are the non-empty flats of `M` and `I_{af}(M)`
    is the ideal generated by

    - all quadratic monomials `x_F x_{F'}` for all incomparable elements
      `F` and `F'`  in the lattice of flats,

    - for all flats `F` and `i \in E \setminus F`

      .. MATH::

          x_F \sum_{i \in F'} x_{F'}

    - and for all `i \in E`

      .. MATH::

        \sum_{i \in F'} (x_{F'})^2.
```
I think this is easier to understand. I would also do the same for the other ideals if you agree.